examples/gke-security-best-practices/istio/istio.md

```diff
@@ -0,0 +1,14 @@
+# Installing Istio on GKE
```

Comment:
There's nothing else about Istio in this solution and it seems unrelated, so I'd drop this directory entirely.